Nativing/accounts/views.py

```diff
@@ -1,3 +1,76 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from .models import *
+from .forms import *
+from django.contrib.auth import logout as auth_logout
+from django.contrib.auth import login as auth_login
+from django.contrib.auth import authenticate
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+def accounts_signup(request):
+    if request.method == "POST":
+        form = SignUpForm(request.POST)
```

Comment:
```suggestion
        form = SignUpForm(request.POST, request.FILES)
```
역시 이미지 파일을 받아오기 위해서는 `request.FILES`가 필요합니다.